db/db.go

```diff
@@ -1466,3 +1466,39 @@ func (db database) ChangeOrganizationDeleteStatus(org_uuid string, status bool)
 	})
 	return ms
 }
+
+func (db database) UpdateOrganizationForDeletion(uuid string) error {
+	updates := map[string]interface{}{
+		"website":     nil,
```

Comment:
@AbdulWahab3181 change all `nil` to empty strings `""`

Reply:
Addressed